app/models/scimitar/schema/base.rb

```diff
@@ -13,7 +13,7 @@ def initialize(options = {})
 
       # Converts the schema to its json representation that will be returned by /SCHEMAS end-point of a SCIM service provider.
       def as_json(options = {})
-        @meta.location = Scimitar::Engine.routes.url_helpers.scim_schemas_path(name: id)
+        @meta.location ||= Scimitar::Engine.routes.url_helpers.scim_schemas_path(name: id)
```

Comment:
Unsure if generating a url here still has any value. I left it in here for backwards compatibility.